src/ConfigFile.c

```diff
@@ -327,7 +333,7 @@ void Ros_ConfigFile_CheckYamlEvent(yaml_event_t* event)
                 case Value_UserLanPort:
 #if defined (FS100) || defined (DX200)
                     // single port, override whatever was configured
-                    *(Ros_UserLan_Port_Setting*)activeItem->valueToSet = CFG_ROS_USER_LAN1;
+                    * (Ros_UserLan_Port_Setting*)activeItem->valueToSet = CFG_ROS_USER_LAN1;
                     Ros_Debug_BroadcastMsg("DX200 or FS100: override to 'USER_LAN1'");
 
 #elif defined (YRC1000) || defined (YRC1000u)
```

Comment:
This is not accurate. YRCu only has one LAN port.

Reply:
We'll fix the `YRC1000u` case when addressing #310.